### PR TITLE
Elo updates

### LINKED ISSUE #872: Card ELO sort option
As a user, I would like to be able to sort my cube by the cards' ELO.

### LINKED ISSUE #873: More Information on Card Pages
As a user, on card pages I would like to see information about the card including:
Price, Foil Price (if it exists), ELO

### LINKED ISSUE #874: View ELO on card modal
As a user, I would like to be able to check the ELO of a card easily.

Can we put it right underneath where we put the price in the context modal? Makes sense since it's a field we can't edit, like price.

### LINKED ISSUE #877: Use Elo for draft bots
The Elo data looks good enough now that I think we're ready to use Elo for our draft bots.

### PR DESCRIPTION
**Pull request by dekkerglen**
Fixes #877, Fixes #874, Fixes #872, Fixes #873

### CODE REVIEW

File: src/util/Draft.js [hunk 1]
```diff
@@ -81,12 +88,13 @@ function botPicks() {
...
 
     ratedPicks.sort((x, y) => {
-      return botRating(botColors, pack[x]) - botRating(botColors, pack[y]);
+      return botRating(botColors, pack[y]) - botRating(botColors, pack[x]);
     });
     arrayShuffle(unratedPicks);
 
     const pickOrder = ratedPicks.concat(unratedPicks);
     pick = pack.splice(pickOrder[0], 1);
+    console.log(pick[0].details.name + ' - ' + botColors);
```
Comment (phulin) [inline src/util/Draft.js]: Take this out

File: src/util/Sort.js [hunk 2]
```diff
@@ -335,6 +336,31 @@ export function getLabels(cube, sort) {
...
+    cube.forEach(function(card, index) {
+      if (card.details.elo) {
+        if (!items.includes(card.details.elo)) {
+          items.push(card.details.elo);
+        }
+      }
+    });
+    return items.sort(function(x, y) {
+      if (!/^\d+$/.test(x) || !/^\d+$/.test(y)) {
+        if (x > y) {
```
Comment (phulin) [inline src/util/Sort.js]: Shouldn't all the elos be integers? Why do you need to do this?

File: src/util/Sort.js [hunk 3]
```diff
@@ -335,6 +336,23 @@ export function getLabels(cube, sort) {
     return labels;
   } else if (sort == 'Unsorted') {
     return ['All'];
+  } else if (sort == 'Elo') {
+    var items = [];
+    cube.forEach(function(card, index) {
```
Comment (RKelln) [inline src/util/Sort.js]: Buckets?
- find elo range from cards (min to max)
- bucket_size = 10 * elo_range / num_cards_in_cube   
- bucket size min 1

Comment (phulin) [reply src/util/Sort.js]: Just so it's written down somewhere, the Elo system should be centered at 1200 with a range of roughly 400-2000, so buckets can be made accordingly.

Comment (dekkerglen) [reply src/util/Sort.js]: I'll do buckets of 50, and just use math to divide it up

**Pull request merged**